<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="7" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="57695" started-at="2018-11-26T09:51:40Z" finished-at="2018-11-26T09:52:38Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="57695" started-at="2018-11-26T09:51:40Z" finished-at="2018-11-26T09:52:38Z">
      <class name="Report.Mann_Early_Warning">
        <test-method status="PASS" signature="setup1()[pri:0, instance:Report.Mann_Early_Warning@f777e8]" name="setup1" is-config="true" duration-ms="14155" started-at="2018-11-26T15:21:40Z" finished-at="2018-11-26T15:21:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup1 -->
        <test-method status="PASS" signature="TCED37101()[pri:1, instance:Report.Mann_Early_Warning@f777e8]" name="TCED37101" duration-ms="8356" started-at="2018-11-26T15:21:55Z" finished-at="2018-11-26T15:22:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED37101 -->
        <test-method status="PASS" signature="TCED37102()[pri:2, instance:Report.Mann_Early_Warning@f777e8]" name="TCED37102" duration-ms="41" started-at="2018-11-26T15:22:03Z" finished-at="2018-11-26T15:22:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED37102 -->
        <test-method status="PASS" signature="TCED37103()[pri:3, instance:Report.Mann_Early_Warning@f777e8]" name="TCED37103" duration-ms="2353" started-at="2018-11-26T15:22:03Z" finished-at="2018-11-26T15:22:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED37103 -->
        <test-method status="PASS" signature="TCED37104()[pri:4, instance:Report.Mann_Early_Warning@f777e8]" name="TCED37104" duration-ms="2261" started-at="2018-11-26T15:22:05Z" finished-at="2018-11-26T15:22:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED37104 -->
        <test-method status="FAIL" signature="TCED37107()[pri:7, instance:Report.Mann_Early_Warning@f777e8]" name="TCED37107" duration-ms="7697" started-at="2018-11-26T15:22:08Z" finished-at="2018-11-26T15:22:15Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Sorted asc] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Sorted asc] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at Report.Mann_Early_Warning.verify_sort_ascending(Mann_Early_Warning.java:258)
	at Report.Mann_Early_Warning.TCED37107(Mann_Early_Warning.java:192)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED37107 -->
        <test-method status="PASS" signature="TCED37108()[pri:8, instance:Report.Mann_Early_Warning@f777e8]" name="TCED37108" duration-ms="11460" started-at="2018-11-26T15:22:15Z" finished-at="2018-11-26T15:22:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED37108 -->
        <test-method status="PASS" signature="TCED37109()[pri:9, instance:Report.Mann_Early_Warning@f777e8]" name="TCED37109" duration-ms="11314" started-at="2018-11-26T15:22:27Z" finished-at="2018-11-26T15:22:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED37109 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:Report.Mann_Early_Warning@f777e8]" name="tearDown" is-config="true" duration-ms="1" started-at="2018-11-26T15:22:38Z" finished-at="2018-11-26T15:22:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Report.Mann_Early_Warning -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
